docs/en/pid-eaa-issuance.rst

```diff
@@ -439,50 +454,152 @@ If the checks defined above are successful the Wallet Instance proceeds with the
+    
+    Refresh Token Flow
 
-Technical Flow
-^^^^^^^^^^^^^^
+.. note::
+  The refresh of a Token may be triggered by different actions (e.g., User deletion of a Digital Credential). In each case, Wallet Instances are supposed to be running and the corresponding cryptographic material unlocked.
 
-If PID/(Q)EAA Providers, supporting this flow, are not able to immediately issue a requested Credential, they MUST provide the Wallet Instance with an HTTP Credential Response cointaining the amount of time to wait before making a new Credential request and an identifier of the deferred issuance transaction (*transaction_id*). The HTTP status code MUST be *202* (see Section 15.3.3 of [:rfc:`9110`]). Below a non-normative example is given.
+**Step 1.** The Wallet Instance MUST create a fresh DPoP Proof JWT and a fresh Wallet Attestation proof of possession for the token request of the PID/(Q)EAA Provider. 
 
-.. code-block:: http
+**Step 2.** To refresh a DPoP-bound Access Token, the Wallet Instance sends a token request using the parameter ``grant_type`` set to ``refresh_token``, including the DPoP header and the OAuth Client Attestation headers.  
+Non-normative example of the token request for a DPoP Access Token using a Refresh Token.
 
-  HTTP/1.1 202 Accepted
+.. code::
+
+  POST /token HTTP/1.1
+  Host: eaa-provider.example.org
+  Content-Type: application/x-www-form-urlencoded
+  DPoP: eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwieCI6IjR2dDhNdEFISmlsMzBDNnpUTmt2c0VVcnlHTEUtQW5BNkc5LV8xa3l5Rk0iLCJ5IjoiTWdiNTFfbjNSRjNtbHNtS3dMd0xtRUFqVmlJM3Q1bTVWNTI2MFA5MzR3RSIsImNydiI6IlAtMjU2In19.eyJqdGkiOiItQndDM0VTYzZhY2MybFRjIiwiaHRtIjoiR0VUIiwiaHR1IjoiaHR0cHM6Ly9yZXNvdXJjZS5leGFtcGxlLm9yZy9wcm90ZWN0ZWRyZXNvdXJjZSIsImlhdCI6MTU2MjI2MjYxOH0.3Tp1ZlZ05PQYeZUHhiZwaQ1etqnwYwoiJHFR_JHb32381lMJL-8o2rE3VZ8X3yuqrGFfCVeP90Ln4J5r8ASIBg
+  OAuth-Client-Attestation: eyJhbGciOiJFUzI1NiIsImtpZCI6IjBiNDk4ZGRlMDkxNzJhZGE3MDFkMDdlYjZmOTg2N2FkIn0.eyJpc3MiOiJodHRwczovL2NsaWVudC5leGFtcGxlLmNvbSIsInN1YiI6IjQ3Yjk4MjM2OTc5MWQwODAwM2E3MjgzZjA1OWNiMGQxIiwibmJmIjoxMzAwODE1NzgwLCJleHAiOjEzMDA4MTkzODAsImNuZiI6eyJqd2siOnsia3R5IjoiRUMiLCJ4IjoiZ1YzSDI0WGQyakhfaWFBd3hkbWlmMm5oeG9uUDhpZTQyQmE5UFhxR0RBcyIsInkiOiJpckZ3b0xabVNZd1FBWHpkZzUxcElPVDdtQ1R3RGtKQWUtdEctVUttYndrIiwiY3J2IjoiUC0yNTYifX19.qlYrkSCkrSDmmQNbbbi6Vj5BHJogOS-y2UErKinhmVUKLLDcBp3C_wasArEHiLnzykLi3hURNJEAAU7v798O6w
+  OAuth-Client-Attestation-PoP: eyJhbGciOiJFUzI1NiJ9.eyJpc3MiOiIgaHR0cHM6Ly9jbGllbnQuZXhhbXBsZS5jb20iLCJhdWQiOiJodHRwczovL2FzLmV4YW1wbGUuY29tIiwianRpIjoiZDI1ZDAwYWItNTUyYi00NmZjLWFlMTktOThmNDQwZjI1MDY0IiwiaWF0IjoxNzIyMjQ5NDQ3LCJleHAiOjE3MjIyNDk3NDd9.8pS0-4QWSl8kfR9FpbS51IQ0BvP0ZbTutbBlVX5V_LIBzC5weCD_mfl339-zVKAqZXc7rAWa3TH2A5SzsimuIA
```

Comment:
missing typ

iat and exp have the same value, they must be one minor than the other